src/pages/addTask/addtask.html

```diff
@@ -0,0 +1,29 @@
+<div class="addTask">
+  <h2>Add new</h2>
+  <form class="form">
+    <div class="form__textinput">
+      <label for="task" class="textinput__label">Enter Task </label>
+      <input type="text" id="task" class="textinput__input" />
+    </div>
+
+    <fieldset class="radiocontainer">
+      <legend>WHEN</legend>
+      <div class="radiocontainer__radiobtn">
+        <div>
+          <input type="radio" value="today" id="today" name="when" />
+          <label for="today">Today</label>
+        </div>
+        <div>
+          <input type="radio" value="tomorrow" id="tomorrow" name="when" />
+          <label for="Tomorrow">Tomorrow</label>
```

Comment:
Should be lowercase, because the for connection to the id should be case-sensitive.